From 38f47899a53b9deb73b092c387834cb91ea8e909 Mon Sep 17 00:00:00 2001
From: Andre McCurdy <armccurdy@gmail.com>
Date: Tue, 23 Apr 2019 13:20:23 -0700
Subject: [PATCH] force legacy -mbe32 mode by default

The Puma6 ARM1176JZ-S core seems to be stuck in legacy BE-32 mode?

  http://infocenter.arm.com/help/topic/com.arm.doc.ddi0333h/Cdfbbchb.html
  http://infocenter.arm.com/help/topic/com.arm.doc.ddi0333h/Cdfbbchb.html#Cdfejcie

Signed-off-by: Andre McCurdy <armccurdy@gmail.com>
---
 gcc/config/arm/arm-cpus.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc/config/arm/arm-cpus.in b/gcc/config/arm/arm-cpus.in
index 545321b..bf03f1b 100644
--- a/gcc/config/arm/arm-cpus.in
+++ b/gcc/config/arm/arm-cpus.in
@@ -225,7 +225,7 @@ define fgroup ARMv5t      ARMv5 thumb
 define fgroup ARMv5e      ARMv5 armv5e
 define fgroup ARMv5te     ARMv5e thumb
 define fgroup ARMv5tej    ARMv5te
-define fgroup ARMv6       ARMv5te armv6 be8
+define fgroup ARMv6       ARMv5te armv6
 define fgroup ARMv6j      ARMv6
 define fgroup ARMv6k      ARMv6 armv6k
 define fgroup ARMv6z      ARMv6
-- 
1.9.1

